CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 1.  [#4217](https://github.com/influxdata/chronograf/pull/4217): Add filestore backed API for protodashboards
 
 ### UI Improvements
+1.  [#4227](https://github.com/influxdata/chronograf/pull/4227): Redesign Cell Editor Overlay
```

Comment:
I figure when the dust settles and before next release it will be easier to describe the changes. Like maybe one entry for the epic rather than the issues?

Reply:
I think its fine to have this one in here, since its more of a precursor to flux rather than having any flux additions in it. I can update the description.